BrainPortal/db/migrate/20250324164153_rename_message_group_id_to_invitation_group_id.rb

```diff
@@ -0,0 +1,27 @@
+class RenameMessageGroupIdToInvitationGroupId < ActiveRecord::Migration[5.0]
```

Comment:
All these checks are unnecessary. The migration you had in the previous commit was already perfectly fine. There is no need to check for the columns and create it if needed, because whent he migration is run, it is guaranteed that the columns will be present. Just revert to the what it was:

```ruby
  def up
    rename_column :messages, :group_id, :invitation_group_id
  end

  def down
    rename_column :messages, :invitation_group_id, :group_id
  end
```